actionview/test/template/log_subscriber_test.rb

```diff
@@ -83,6 +90,36 @@ def test_render_partial_with_implicit_path
     end
   end
 
+  def test_render_partial_with_cache_missed
+    Rails.stub(:root, File.expand_path(FIXTURE_LOAD_PATH)) do
+      def @view.view_cache_dependencies; []; end
+      def @view.fragment_cache_key(*); 'ahoy `controller` dependency'; end
+      set_cache_controller
+
+      @view.render(partial: "test/cached_customer", locals: { cached_customer: Customer.new("david") })
+      wait
+
+      assert_equal 1, @logger.logged(:info).size
+      assert_match(/Rendered test\/_cached_customer\.erb (.*) \[cache miss\]/, @logger.logged(:info).last)
+    end
+  end
+
+  def test_render_partial_with_cache_hitted
+    Rails.stub(:root, File.expand_path(FIXTURE_LOAD_PATH)) do
+      def @view.view_cache_dependencies; []; end
+      def @view.fragment_cache_key(*); 'ahoy `controller` dependency'; end
+      set_cache_controller
+
+      @view.render(partial: "test/cached_customer", locals: { cached_customer: Customer.new("david") })
```

Comment:
Think it's okay if we merge the above assertion into here and then kill the above test. We're testing the integration here after all, so fine with some lifecycle to it.